### PR TITLE
Error in configuration attributes

### PR DESCRIPTION
**Pull request by hmunfru**
Bug in the configuration of attributes in the PaaS Manager
@jframos @jesuspg 
https://jira.tid.es/browse/CLAUDIA-5006

### CODE REVIEW

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java [hunk 1]
```diff
@@ -368,4 +368,26 @@ public void testResolveMacroNamesIpAll() {
...
+        ProductInstallatorSdcImpl installator = new ProductInstallatorSdcImpl();
+        com.telefonica.euro_iaas.sdc.model.Attribute att = new com.telefonica.euro_iaas.sdc.model.Attribute("key",
+            "default", "desc", "IP");
+        com.telefonica.euro_iaas.sdc.model.Attribute newAtt = installator.resolveAttributeTypeValue(att, tierInstance,
+            environmentInstance);
+        assertEquals("default", newAtt.getValue());
+
+    }
+
+    @Test
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java]: Add a test testResolveMacroRealValue with correct IP(xxx)

**Comment by flopezag** [comment]
You are decreased the coverage...

**Comment by flopezag** [comment]
The description of the PR does not include the person who should check it, the description of the changes and the jira issue to be associated to it.

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java (hunk 1 above)
Comment (hmunfru) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java]: they already exist

**Comment by hmunfru** [comment]
increasing coverage

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java (hunk 1 above)
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImplTest.java]: NTC

**Comment by flopezag** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**